sympy/core/tests/test_args.py

```diff
@@ -468,8 +468,8 @@ def test_sympy__sets__sets__Complement():
 
 def test_sympy__sets__sets__SymmetricDifference():
     from sympy.sets.sets import FiniteSet, SymmetricDifference
-    assert _test_args(SymmetricDifference(FiniteSet(1, 2, 3), \
-           FiniteSet(2, 3, 4)))
+    assert _test_args(SymmetricDifference(FiniteSet(1, 2, 3),
```

Comment:
It wasn't pure formatting though, I have added `args` tests for  ComplexPlane Class, in that file.
I read somewhere in mailing list discussion by **jcrist** , It's good to make formatting changes around the code we are playing with. Isn't it?